Documentation/op-guide/grafana.json

```diff
@@ -341,7 +340,7 @@
                         "steppedLine": false,
                         "targets": [
                             {
-                                "expr": "etcd_mvcc_db_total_size_in_bytes{job=\"$cluster\"}",
+                                "expr": "etcd_debugging_mvcc_db_total_size_in_bytes{job=\"$cluster\"}",
```

Comment:
@gyuho has the "debugging" namespace recently been removed? If so @serathius we should make sure this is the same in the mixin, or maybe even configurable, so the mixin is still useful for older versions of etcd.

Reply:
@brancz Yes https://github.com/coreos/etcd/pull/9819/files. We should use `etcd_mvcc_db_total_size_in_bytes` instead. `etcd_debugging_mvcc_db_total_size_in_bytes` will still be served in 3.4 for backward compatibilities but will be dropped in 3.5.

Reply:
Thanks for clarifying @gyuho in that case we should make the metric name configurable in the mixin @serathius. Could you add a `etcdDbTotalSizeMetric` configuration under the top level `_config` to configure this metric name and have it default to `etcd_mvcc_db_total_size_in_bytes`. Then downstream users can configure it to the metric name they need. Once 3.5 is out we can then drop the config entirely. Sorry for all the trouble @serathius , I promise once we have that down this lgtm :slightly_smiling_face: .